esphome/components/ams5915/ams5915.h

```diff
@@ -0,0 +1,130 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/core/hal.h"
+#include "esphome/components/i2c/i2c.h"
+#include "esphome/components/sensor/sensor.h"
+
+namespace esphome {
+namespace ams5915 {
+enum Transducer {
```

Comment:
Add some spacing around things :wink: 
```suggestion

enum Transducer {
```